src/AppserverIo/Appserver/ServletEngine/RequestHandler.php

```diff
@@ -186,6 +186,7 @@ public function run()
             }
 
         } catch (\Exception $e) {
+            ErrorUtil::singleton()->flattenExceptionBacktrace($e);
```

Comment:
Should be moved from `RequestHandler::run()` to inside `ErrorUtil::fromException()` method as this method can be invoke from somewhere else to.

Reply:
Updated the pull request.